src/components/Title.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { StyleSheet } from "react-native";
+import { useTheme, Title as RNPTitle } from "react-native-paper";
```

Comment:
`RNPTitle` ? What does that mean?

Reply:
As suggested by @guilbill I just move this to `PaperTitle`